tests/WpfCalculator.Tests/WpfCalculator.Tests.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Xpf.Sdk/1.5.3">
+
+    <PropertyGroup>
+        <TargetFramework>net8.0</TargetFramework>
+        <IsPackable>false</IsPackable>
+        <IsTestProject>true</IsTestProject>
+        <!-- Important to disable automatic Avalonia XPF initialization, because tests will do it manually -->
+        <DisableAutomaticXpfInit>true</DisableAutomaticXpfInit>
+    </PropertyGroup>
+
+    <ItemGroup>
+<!--        <PackageReference Include="Avalonia.Headless.XUnit" Version="$(XpfAvaloniaVersion)" />-->
+        <PackageReference Include="Avalonia.Headless.XUnit" Version="11.3.999-cibuild0059922-alpha" />
```

Comment:
TODO: once new XPF version is released with these changes, this sample should be updated to use `$(XpfAvaloniaVersion)`.
Meanwhile, nightly build of this package can be used.